### ISSUE TITLE
cannot complete bothan-deploy due to name error

### ISSUE THREAD

**Issue by quadrophobiac** [theodi/bothan-deploy#10]
There is clearly something amiss with Bothan Deploy. All attempts to deploy a bothan instance via the one-button deploy fail due to `Application name <$name> unavailable` - see screenshots below

Steps to Reproduce
1. navigate to bothan deploy https://deploy.bothan.io/
2. fill in any suitably unique combination of characters for the name
3. you should see this error
![screen shot 2017-08-17 at 11 51 20](https://user-images.githubusercontent.com/1039017/29409047-78d807da-8342-11e7-8113-81f061789722.png)
and this in the console
![screen shot 2017-08-17 at 11 51 26](https://user-images.githubusercontent.com/1039017/29409061-81befeda-8342-11e7-9114-56d5e8cca20b.png)

**Comment by pezholio** [comment]
Yeah, looks like the Heroku API has changed. Need to pass the version and MIME type in the headers. See:

https://api.heroku.com/apps/hello
